sql/src/test/java/io/crate/operation/operator/any/AnyEqOperatorTest.java

```diff
@@ -104,4 +104,10 @@ public void testNormalizeSymbol() throws Exception {
         assertNormalize("42 = ANY([42])", isLiteral(true));
         assertNormalize("42 = ANY([41, 43, -42])", isLiteral(false));
     }
+
+    @Test
+    public void testExceptionForwarding() throws Exception {
+        expectedException.expect(IllegalArgumentException.class);
+        anyEq(1, "bar");
```

Comment:
pls add an expected exception message (and re-name the test method to explain what this test is doing). currently I have no glue what is tested here